DrinkingPub/Order.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Vsite.Oom.DrinkingPub
+{
+    public class Order
+    {
+        private Dictionary<string, int> OrderItems = new();
+
+        public Order() { }
+
+        public Dictionary<string, int> GetOrderItems()
```

Comment:
Ova metoda vraća referencu na interni objekt pa stoga možete u metodi `Table.TotalTotalToPay`, iznad unutarnje petlje `foreach` napisati naredbu:
`orders.GetOrderItems().Add(...);`
koja će promijeniti sadržaj narudžbe izvan klase `Order`. Da se to onemogući, može se definirati svojstvo:
`IEnumerable<KeyValuePair<string, int>> Items => OrderItems.AsEnumerable();`